tests/Symbol/Reflector/PhpStormStubsReflectorTest.php

```diff
@@ -98,6 +98,20 @@ public static function provideClasses(): iterable
             'MongoInsertBatch',
             'MongoDeleteBatch',
         );
+
+        yield from self::createDataSetForInternalSymbols(
+            'HP 8.4 new class-like (added or modified)',
```

Comment:
typo?

```suggestion
            'PHP 8.4 new class-like (added or modified)',
```

Reply:
yes!

Reply:
FYI I'll probably won't have time to look into before new year, kinda under water at work and then I'll benefit from being more off for the festivities